helm-charts/splunk-otel-collector/templates/config/_otel-k8s-cluster-receiver-config.tpl

```diff
@@ -94,6 +94,21 @@ processors:
         key: {{ .name }}
         value: {{ .value }}
       {{- end }}
+      # Extract "container.image.tag" attribute from "container.image.name" here until k8scluster
+      # receiver does it natively.
+      - key: container.image.name
+        pattern: ^(?P<container_image_name>[^\:]+)(?:\:(?P<container_image_tag>.*))?
```

Comment:
Should you be using a temp prefix in the unlikely event these are stepping on custom attributes?

Reply:
You're right. Thanks! Let me update it

Reply:
done